src/con_duct/pprint_json.py

```diff
@@ -65,10 +66,14 @@ def humanize_data(data: Any, formatter: SummaryFormatter) -> Any:
 def pprint_json(args: argparse.Namespace) -> int:
     """
     Prints the contents of a JSON file using pprint.
+
+    Handles both standard JSON files and JSON Lines (usage.jsonl) files.
     """
     try:
-        with open(args.file_path, "r") as file:
-            data = json.load(file)
+        if is_jsonl_file(args.file_path):
+            data: Any = load_usage_file(args.file_path)
```

Comment:
```suggestion
            data = load_usage_file(args.file_path)
```

IDT this annotation communicates anything

Reply:
😆

OK wow - can we maybe make it useful then? `data: list[Any]` or even more accurate